test/NosCore.Tests/BazaarTests/CScalcPacketHandlerTests.cs

```diff
@@ -46,11 +47,11 @@ namespace NosCore.Tests.BazaarTests
     public class CScalcPacketHandlerTest
     {
         private static readonly ILogger _logger = Logger.GetLoggerConfiguration().CreateLogger();
-        private Mock<IBazaarHttpClient> _bazaarHttpClient;
-        private CScalcPacketHandler _cScalcPacketHandler;
-        private Mock<IGenericDao<IItemInstanceDto>> _itemInstanceDao;
-        private Mock<IItemProvider> _itemProvider;
-        private ClientSession _session;
+        private Mock<IBazaarHttpClient>? _bazaarHttpClient;
+        private CScalcPacketHandler? _cScalcPacketHandler;
+        private Mock<IGenericDao<IItemInstanceDto>>? _itemInstanceDao;
+        private Mock<IItemProvider>? _itemProvider;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_itemProvider' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262030)